transformers/transformers_simple.py

```diff
@@ -0,0 +1,104 @@
+"""
+Optuna example for fine-tuning a BERT-based text classification model on the IMDb dataset
+with hyperparameter optimization using Optuna. In this example, we fine-tune a lightweight
+pre-trained BERT model on a small subset of the IMDb dataset to classify movie reviews as
+positive or negative. We optimize the validation accuracy by tuning the learning rate, batch size
+and number of training epochs.
+"""
+
+from datasets import load_dataset
+import evaluate
+import torch
+
+from transformers import AutoModelForSequenceClassification
+from transformers import AutoTokenizer
+from transformers import set_seed
+from transformers import Trainer
+from transformers import TrainingArguments
+
+
+set_seed(42)
+
+
+device = torch.device("cpu")
+
+
+raw_dataset = load_dataset("imdb")
+
+raw_train = raw_dataset["train"].shuffle(seed=42).select(range(1000))
```

Comment:
`load_dataset("imdb")` contains three splits, where, we don't need `unsupervised` split. Could you follow my previous suggestion?
```suggestion
train_dataset = load_dataset("imdb", split="train").shuffle(seed=42).select(range(1000))
```

Reply:
Updated as discussed.